Jenkinsfile

```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env groovy
 
-boolean main = env.BRANCH_NAME == "main"
-boolean beta = env.BRANCH_NAME == "private-beta"
-def bucket   = main ? "production"
-             : beta ? "staging"
+boolean isProduction = env.BRANCH_NAME == "production"
+boolean isBeta = env.BRANCH_NAME == "beta"
+boolean isDevelop = env.BRANCH_NAME = "develop"
```

Comment:
does jenkins provide these automatically? if so, that's great

Reply:
I assume, I didn't changed anything in the Jenkins UI already, will need to check.

Reply:
It did before, why should it change?